stdlib/argparse.pyi

```diff
@@ -161,17 +161,12 @@ class ArgumentParser(_AttributeHolder, _ActionsContainer):
             add_help: bool = True,
             allow_abbrev: bool = True,
         ) -> None: ...
-    # The type-ignores in these overloads should be temporary.  See:
-    # https://github.com/python/typeshed/pull/2643#issuecomment-442280277
```

Comment:
The `type: ignore`s have been here for 5 years now, time to stop claiming that they're "temporary" 😄